assets/src/edit-story/app/highlights/quickActions/useQuickActions.js

```diff
@@ -172,14 +173,18 @@ const useQuickActions = () => {
     handleFocusAnimationPanel,
     handleFocusLinkPanel,
     handleFocusPageBackground,
+    handleFocusTextColor,
+    handleFocusFontPicker,
     handleFocusTextSetsPanel,
     handleFocusStylePanel,
   } = useMemo(
     () => ({
       handleFocusAnimationPanel: handleFocusPanel(states.ANIMATION),
       handleFocusLinkPanel: handleFocusPanel(states.LINK),
       handleFocusPageBackground: handleFocusPanel(states.PAGE_BACKGROUND),
-      handleFocusTextSetsPanel: handleFocusPanel(states.TEXT),
+      handleFocusTextSetsPanel: handleFocusPanel(states.TEXT_SET),
```

Comment:
it was a little confusing having the `TEXT` state by for the text sets panel, so I updated it to be `TEXT_SET` instead now that we also have text_color and font (which live in the text panel)